index.bs

```diff
@@ -1610,7 +1660,7 @@ Issue: handle String / Number / etc. wrapper objects specially?
 <div algorithm>
 
 To <dfn>serialize as a remote value</dfn> given a |value|, a |max depth|,
-|node details|, a |set of known objects| and a |realm|:
+a |node details|, an |ownership|, a |serialization internal map| and a |realm|:
```

Comment:
Maybe using `|ownership type|` or similar everywhere would be clearer? Minor nit though.

Reply:
`|ownership type|` seems more readable, done, thanks!